serde/src/test/org/apache/hadoop/hive/serde2/objectinspector/TestThriftObjectInspectors.java

```diff
@@ -176,6 +183,11 @@ public void testThriftSetObjectInspector() throws Throwable {
       assertEquals("setString", soi.getStructFieldData(s, fields.get(1)));
 
       // sub fields
+      Collections.sort(fields, new Comparator<StructField>() {
```

Comment:
Similarly, this could be shortened to `Comparator.comparing(StructField::getFieldObjectInspector`.

However, I'm also not clear why it would need to be resorted by a different key for the next assertions.